assets/src/edit-story/components/richText/karma/multiSelection.karma.js

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { waitFor } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { initHelpers } from './_utils';
+
+describe('CUJ: Editor Can Style Text', () => {
+  describe('Action: Styling text in multiple text fields', () => {
+    const data = {};
+
+    const {
+      getTextContent,
+      addInitialText,
+      selectBothTextFields,
```

Comment:
Would this helpers make more sense as general helpers? E.g. `addTextFields([{'text1'},{'text2']])` and `selectTextFields()` or `selectTextFields(2)` or sth like that? Or are these helpers usable for these tests only? Perhaps it's fine in this context though, just feels a bit unusual by name.

Reply:
I felt that you'd often want many different custom settings, that it was hard to generalize - positioning the elements with enough space between, that they'd never overlap (even the rotation handle shouldn't overlap), was one such annoyance. If needed elsewhere, I'd recommend picking them up from here and generalizing.